txtorcon/torconfig.py

```diff
@@ -423,6 +424,11 @@ class EphemeralHiddenService(object):
     https://gitweb.torproject.org/torspec.git/tree/control-spec.txt#n1295
     '''
 
+    @classmethod
+    def _is_valid_keyblob(cls, key_blob_or_type):
+        return (isinstance(key_blob_or_type, str) and
+                re.match(r'[^ :]+:[^ :]+$', key_blob_or_type))
```

Comment:
I prefer this approach to not break stuff rather than the other - not at least without a warning :P

I also prefer using what Twisted already offers or maybe creating this abstract helper in `util` that uses `nativeString` underneath, but can be easily modified if one day is necessary.

(That's just my opinion as a user. You know better than me.)

Questions:

- What is this kind of change supposed to do? Always send unicode to Tor?
- Is it something that should be done in the entire code base?
- Is the whole API supposed to receive/return unicode?

Thanks!